### ISSUE TITLE
Axe of a Thousand Metals Special attacks non-functional...

### ISSUE THREAD

**Issue by Bluepugs** [BobMowzie/MowziesMobs#265]
Hiya, playing the latest version of MC Eternal (1.3.7.1) on multiplayer.

Axe worked fine, until Spongeforge were installed - I believe it is the problem, at least. Now the axe places the animations when right/shift-rightclicking but no actual attack is registered against enemies, nor is there the shockwave.

Is this fixable? Using spongeforge-1.12.2-2838-7.2.1-RC4016 version of spongeforge, as recommended by the MC Eternal discord. Can i help debug the issue with any particular log?

**Comment by BobMowzie** [comment]
Ooh, hm. Interaction issues are always trickiest. If you disable Spongeforge, do the attacks work again? And does the latest MC Eternal use Mowzie’s Mobs 1.5.8?

**Comment by Bluepugs** [comment]
Yeah, playing in singleplayer without spongeforge works, and the axe in full functionality was working before spongeforge on the multiplayer server.

Does look like MC Eternal does use Mowzie Mobs 1.5.8

**Comment by BobMowzie** [comment]
That helps. Thanks.

**Comment by Bluepugs** [comment]
Please let me know if there's any additional debugging I can do to help.

**Comment by 71vl17** [comment]
Craft To Exile [Dissonance] [v. 2.9.3](https://www.curseforge.com/minecraft/modpacks/crafttoexiledissonance/files/3242087) (MC 1.15.2) similar issue - axe doesn't do special attacks, both in SSP/SMP
